linuxkit.yml

```diff
@@ -6,6 +6,7 @@ init:
   - linuxkit/runc:d5cbeb95bdafedb82ad2cf11cff1a5da7fcae630
   - linuxkit/containerd:b6ffbb669248e3369081a6c4427026aa968a2385
   - linuxkit/ca-certificates:67acf038c44bb191ebb704ec7bb39a1524052cdf
+  - linuxkit/wireguard-tools:26fe3d38455f2d441549e3c54bdec1b26ac819b8
```

Comment:
Yes; can add an example later.

We could use the packages from Alpine testing, we currently do a similar thing with one package from community see https://github.com/linuxkit/linuxkit/blob/master/tools/alpine/Dockerfile#L28 - if you add it to here as well it is then easy to add to any package.